CIP-0103/README.md

```diff
@@ -0,0 +1,86 @@
+---
+CIP: 103
+Title: Web-Wallet Bridge - Bulk transaction signing
+Category: Wallets
+Status: Proposed
```

Comment:
```suggestion
Status: Active
```
May want to consider merging as active, due to existing implementers

Reply:
Hi, I guess I missed the slot for the meeting but yes, I would consider the Acceptance Criteria fulfilled.